.changeset/moody-numbers-move.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": minor
+---
+
+Create a function to filter out answer data from categorizer's widget options and add to its widget export
```

Comment:
Since this is where we introduce this concept, I'm going to suggest a few small wording improvements. Feel free to edit further. :) 

```suggestion
Introduce a widget export function to filter out scoring data from widget options. Implement this function for the categorizer widget.
```